mne_connectivity/spectral/time.py

```diff
@@ -43,16 +46,31 @@ def spectral_connectivity_time(data, names=None, method='coh', indices=None,
             * 'sxy' : Cross-spectrum
 
         By default, the coherence is used.
+    average : bool
+        Average connectivity scores over Epochs. If True, output will be
+        an instance of ``SpectralConnectivity`` , otherwise
+        ``EpochSpectralConnectivity``.
     indices : tuple of array | None
         Two arrays with indices of connections for which to compute
         connectivity. I.e. it is a ``(n_pairs, 2)`` array essentially.
         If None, all connections are computed.
     sfreq : float
         The sampling frequency.
-    foi : array_like | None
-        Extract frequencies of interest. This parameters should be an array of
-        shapes (n_foi, 2) defining where each band of interest start and
-        finish.
+    fmin : float | tuple of float
+        The lower frequency of interest. Multiple bands are defined using
+        a tuple, e.g., (8., 20.) for two bands with 8Hz and 20Hz lower freq.
+        If None the frequency corresponding to an epoch length of 5 cycles
+        is used.
+    fmax : float | tuple of float
+        The upper frequency of interest. Multiple bands are dedined using
+        a tuple, e.g. (13., 30.) for two band with 13Hz and 30Hz upper freq.
```

Comment:
Can you add a description of what `fmax is None` will do?

Reply:
Currently, `fmax is None` will set `fmax=sfreq/2`, which is a little bit problematic if `sfreq` is not given as it defaults to the original default value `2*numpy.pi`.

Reply:
`fmin=None` is also a bit problematic as setting it to zero is not an option. Even the current default of 1Hz will cause the regression test to fail since the test data is shorter than the wavelets required for time-frequency decomposition.

Reply:
We should probably make sure that if we allow `None`, that this leads to a good choice of `fmin` and `fmax`. If we cannot come up with anything, we should probably not allow it and force a choice of `fmin` and `fmax`. That again circles back to probably having two functions for different methods.

Reply:
I agree, for me it's a bit hard to see a case where someone would not want to specify `fmin` and `fmax`.

Reply:
`fmin` now defaults to a frequency corresponding to five cycles in the data. `fmax` is set to Nyquist frequency. This is similar to `spectral_connectivity_epochs`.